frontend/test/pytest/test_mitigation.py

```diff
@@ -142,10 +145,22 @@ def circuit(x):
     def mitigated_function(args):
         return catalyst.mitigate_with_zne(circuit, scale_factors=scale_factors)(args)
 
-    with pytest.raises(ValueError, match="The scale factors must be positive odd integers:"):
+    with pytest.raises(
+        ValueError, match=".*Only odd positive integers are allowed in scale_factors"
+    ):
         mitigated_function(0.1)
 
 
+@pytest.mark.parametrize("scale_factors", [[1.0, 3.0], [complex(1, 0), complex(3, 0)]])
```

Comment:
Can we also add the real-world case discussed in slack where `scale_factors=jax.numpy.array([1,3])`? Thanks!